base_station/gui/src/static/rover_msgs.json

```diff
@@ -1203,19 +1233,18 @@
         "NavStatus",
         "Obstacle",
         "Odometry",
-        "OpenLoopRAMotor",
+        "RAPosition",
         "PDBData",
-        "PIDConstants",
-        "ProximitySensors",
         "RGB",
         "RGBFrame",
         "RGBLED",
         "SAMotors",
         "SAPosition",
+        "ScoopLimitSwitchEnabled",
         "SensorPackage",
         "Sensors",
         "Servo",
-        "ServoCmd",
+        "ServoCmds",

```

Comment:
WHAT'S THIS? THE LCM? the lcm is ServoCmd

Reply:
This part of rover msgs is high key super unnecessary anyways

Reply:
can you get rid of Servo while youre at it